pages/index.tsx

```diff
@@ -51,6 +52,17 @@ const { Title, Text } = Typography;
 const Option = Select.Option;
 
 const FEATURED_STOREFRONTS_URL = process.env.FEATURED_STOREFRONTS_URL as string;
+const WHICHDAO = process.env.NEXT_PUBLIC_WHICHDAO as string;
+const DAO_LIST_IPFS = process.env.NEXT_PUBLIC_DAO_LIST_IPFS || "https://ipfs.cache.holaplex.com/bafkreidnqervhpcnszmjrj7l44mxh3tgd7pphh5c4jknmnagifsm62uel4";
+
```

Comment:
```suggestion
const APPROVED_SUBDOMAINS_URL = process.env.NEXT_PUBLIC_APPROVED_SUBDOMAINS_URL;
```

No default. If not set then pull everyting.